eve/workers/pod-builder/pod.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: worker
```

Comment:
Let's have a meaningful name here - `build-worker` for example.

Reply:
as discussed we will keep this name until we get more info from releng